libs/lanforge/lf_tests.py

```diff
@@ -1,19 +1,20 @@
 #########################################################################################################
 # Used by Nightly_Sanity
-# This has different types of tests like Single client connectivity, Single_Client_EAP, testrail_retest
+# This has different types of old_pytest like Single client connectivity, Single_Client_EAP, testrail_retest
```

Comment:
old_pytest does not mean anything to the rest of the community

Reply:
sure i will cleanup from here